mmselfsup/datasets/transforms/processing.py

```diff
@@ -468,6 +536,14 @@ def _rotate(self, img: np.ndarray) -> List[np.ndarray]:
         ]
 
     def transform(self, results: dict) -> dict:
+        """Apply rotation augmentation to the given image.
```

Comment:
```suggestion
        """Apply rotation augmentation to the given image and add ``rot_labels`` to the results.
```

Reply:
The ``added keys`` already exists in the docstring of this class.